tests/screenshot/apps/components/DayPicker.js

```diff
@@ -26,6 +26,25 @@ const DayPickerTests = [
 		component: <DayPicker disabled selected={1} />
 	},
 
+	// *************************************************************
+	// locale = 'es-ES', [QWTC-637]
+	{
+		locale: 'es-ES',
+		component: <DayPicker />
+	},
+	{
+		locale: 'es-ES',
+		component: <DayPicker selected={1} />
+	},
+	{
+		locale: 'es-ES',
+		component: <DayPicker disabled />
+	},
+	{
+		locale: 'es-ES',
+		component: <DayPicker disabled selected={1} />
+	},
+
```

Comment:
```suggestion
```
In 'es-ES' region, <DayPicker selected={1}> causes TypeError. I think it would be better to add this ss test after fixing the DayPicker issue.

Reply:
It's my mistake. Thanks for your help! It will be added after fixing issue.